alerter.c

```diff
@@ -1,16 +1,22 @@
 #include <stdio.h>
 #include <assert.h>
 
+#define PRODUCTIONCODE_ON   1
+#define PRODUCTIONCODE_OFF  0
+#define ENABLE_PRODUCTION_CODE  PRODUCTIONCODE_OFF
 int alertFailureCount = 0;
 
 int networkAlertStub(float celcius) {

```

Comment:
stub and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.

Reply:
We cant able to add new *.c files. So maintain both in same file. Correction done.

Reply:
If the yml file can be updated for this reorganization, then please add.

Reply:
Ok